### ISSUE TITLE
Extension crashes with Gnome 3.28

### ISSUE THREAD

**Issue by ghost** [mymindstorm/gnome-volume-mixer#5]
When I run this extension and run the `sudo journalctl -p 3 -xb` command, I get the following error. `GLib-CRITICAL: g_hash_table_foreach_remove_or_steal: assertion'version == hash_table->version' failed`

**Comment by mymindstorm** [comment]
What is your shell version and distro? Does this occur when the extension is disabled? Could you get more detailed logs from `journalctl /usr/bin/gnome-shell`?

**Comment by Sztoppolino** [comment]
Hi!
I have the same issue. I installed this extension, but I couldn't use it, because it indicated an error. So I opened Gnome Looking Glass and I found an error description:

`TypeError: (intermediate value)._hiddenItem.set_height is not a function`

In the source code "this._hiddenItem.set_height(0);" exist on line 41. I made sure that this line was the problem, because when I deleted it, gnome didn't indicate error (however the extension was unusable as I expected...)

I think that this is a compatibility issue, because I use an older Gnome Shell (v3.28). 

So my question is: How can I use this extension under Gnome 3.28?
